### PR TITLE
Ethereum in activity table

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-193
Screenshot from 2022-10-05 15-41-21

### CODE REVIEW

File: packages/frontend/src/pages/scaling-activity/props/getActivityView.ts [hunk 1]
```diff
@@ -41,3 +47,28 @@ export function getActivityViewEntry(
     transactionsWeeklyCount,
   }
 }
+
+function getEthereumActivityViewEntry(
+  ethereum: Ethereum,
```
Comment (sz-piotr) [inline packages/frontend/src/pages/scaling-activity/props/getActivityView.ts:52]: Why do we need to pass this all the way from the config package? I think having `name: 'Ethereum'` and `slug: 'ethereum'` hardcoded will be much cleaner

File: packages/frontend/src/pages/scaling-activity/view/ActivityView.tsx [hunk 2]
```diff
@@ -48,9 +58,20 @@ export function ActivityView({ items, className }: ActivityViewProps) {
...
+  const rows: RowConfig<ActivityViewEntry> = {
+    getProps: (entry) =>
+      entry.name === 'Ethereum'
+        ? {
+            className: 'accent',
+          }
+        : {},
+  }
+
+  console
```
Comment (sz-piotr) [inline packages/frontend/src/pages/scaling-activity/view/ActivityView.tsx:70]: Hello :)

File: packages/frontend/src/components/ProjectLink.tsx [hunk 3]
```diff
@@ -19,33 +19,33 @@ export function ProjectLink({ project, type }: Props) {
       : `/bridges/projects/${project.slug}`
   return (
     <>
-      <a className="ProjectLink" href={href}>
+      <a className="ProjectCell--link" href={href}>
```
Comment (sz-piotr) [inline packages/frontend/src/components/ProjectLink.tsx:22]: If you are not planning on rewriting the styles of `ProjectLink` in tailwind I would just leave them untouched.

File: packages/frontend/src/components/EthreumCell.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <>
+      <div className="ProjectCell">
+        <img
+          className="ProjectCell-Icon"
+          src={`/icons/${props.project.slug}.png`}
+          alt={`${props.project.name} logo`}
+        />
+        {props.project.name}
+      </div>
```
Comment (sz-piotr) [inline packages/frontend/src/components/EthreumCell.tsx:20]: Please style all new components using tailwind

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/EthreumCell.tsx (hunk 4 above)
Comment (michalsidzej) [reply packages/frontend/src/components/EthreumCell.tsx:20]: Oooh, ok, so the SCSS is just a leftover from old days?

Comment (michalsidzej) [reply packages/frontend/src/components/EthreumCell.tsx:20]: Ethereum cell is sharing all the styling with ProjectLink tho, excluding the hover effect, looks to me like a good idea to use SCSS to keep it consistent here.

Comment (sz-piotr) [reply packages/frontend/src/components/EthreumCell.tsx:20]: > Ethereum cell is sharing all the styling with ProjectLink tho, excluding the hover effect, looks to me like a good idea to use SCSS to keep it consistent here.

I'll let it slide now, but I'm going to definitely enforce it more after I implement the new designs and reduce SCSS use even more

File: packages/frontend/src/build/config/common.ts [hunk 5]
```diff
@@ -18,4 +18,5 @@ export const common = {
   layer2s,
   bridges,
   tokens: tokenList,
+  ethereum,
```
Comment (sz-piotr) [inline packages/frontend/src/build/config/common.ts:21]: This is not exported now, right?

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/table/EthereumCell.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+interface EthereumCellProps {
+  project: {
+    slug: string
+    name: string
+  }
+}
+
+export function EthereumCell({ project }: EthereumCellProps) {
+  return (
+    <>
```
Comment (sz-piotr) [inline packages/frontend/src/components/table/EthereumCell.tsx:12]: The fragment is unnecessary :)

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**